pkg/scripts/stack.sh

```diff
@@ -1,124 +1,234 @@
 #!/usr/bin/env bash
 
-# ``stack.sh`` is an opinionated BigchainDB developer installation.  It
-# installs and configures **BigchainDb Server**, **Tendermint Server**,
-# **MongoDB**
-
-# Print the commands being run so that we can see the command that triggers
-# an error.  It is also useful for following along as the install occurs.
-set -o xtrace
+set -o nounset
 
 # Make sure umask is sane
 umask 022
 
-# Keep track of the stack.sh directory
-TOP_DIR=$(cd $(dirname "$0") && pwd)
-BASE_DIR=${TOP_DIR}/../..
-
-# Check for uninitialized variables, a big cause of bugs
+# defaults
+stack_branch=${STACK_BRANCH:="master"}
+stack_size=${STACK_SIZE:=4}
+stack_type=${STACK_TYPE:="docker"}
+stack_type_provider=${STACK_TYPE_PROVIDER:=""}
+tm_version=${TM_VERSION:="0.19.2"}
```

Comment:
Maybe just do a project-wide search for "0.19.2" and change it to 0.19.3.

Reply:
I changed it to `0.19.2` but Tendermint has not tagged a `0.19.3` image on Docker Hub, so I reverted back to `0.19.2`.